src/Extensions/Activities/Internal/ActivityLoggerHelpers.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Omex.Extensions.Activities
+{
+	internal static class ActivityLoggerHelpers
+	{
+		public static StringBuilder AppendObjStart(this StringBuilder builder) => builder.Append('{');
+
+		public static StringBuilder AppendObjEnd(this StringBuilder builder) => builder.Append('}');
+
+		public static StringBuilder AppendArrayStart(this StringBuilder builder) => builder.Append('[');
+
+		public static StringBuilder AppendArrayEnd(this StringBuilder builder) => builder.Append(']');
+
+		public static StringBuilder AppendSeparator(this StringBuilder builder) => builder.Append(',');
+
+		public static StringBuilder AppendParamName(this StringBuilder builder, string name) =>
+			builder.Append(name).Append(':');
```

Comment:
Should there be any sanity check on name here?
I guess we want to keep it as lightweight as possible but worth considering if even a null check is warranted - or is the assumption that anything using Activities will be using the latest compiler flags/version which will perform null guards?

Reply:
I don't think it's needed for two reasons:
1. we are enforcing nullability so there is no proper way of passing nulls there.
2. `Append` actually fine with `null` it's equivalent of adding an empty string https://sharplab.io/#v2:C4LgTgrgdgNAJiA1AHwAICYAMBYAUBgRj1UwAJUCA6AFQFMAPYAbjzwDcBDMUgI1IF5SUWgHdSAZWBgAllADmAIQjSANnFpgAFAEoW+ApgD8pDgKEQVKvRQCcmgEQAlWgGcLoe6US9KAQQAO/rRQcJoc2jQA9pIy8jq6QA==